content/en/tracing/trace_explorer/trace_view.md

```diff
@@ -21,23 +21,86 @@ further_reading:
   text: "Understand how to read a Datadog Trace"
 ---
 
-View an individual [trace][1] to see all of its [spans][2] and associated metadata. Each trace can be viewed either as a flame graph or as a list (grouped by [service][3] or host).
+{{< img src="tracing/trace_view/trace_view.png" alt="Trace View" style="width:90%;">}}
```

Comment:
Minor comment, but I think we could potentially crop out the Datadog left-navigation and/or the page behind the actual trace view. I think that would help make this screenshot less overwhelming.